ui/src/components/pages/Namespace/partials/PipelineCard/index.tsx

```diff
@@ -259,12 +257,12 @@ export function PipelineCard({
                   }}
                 >
                   <span style={{ marginLeft: "1rem" }}>
-                    Pipeline Pausing...
+                    {statusPayload?.spec?.lifecycle?.desiredPhase === PAUSED
+                      ? "Pipeline Pausing..."
+                      : "Pipeline Resuming..."}
                   </span>
                   <span style={{ marginLeft: "1rem" }}>
-                    {statusData?.pipeline?.status?.lastUpdated
-                      ? timeAgo(statusData?.pipeline?.status?.lastUpdated)
-                      : ""}
+                    {timeAgo(new Date().toISOString())}
```

Comment:
Shouldn't this be timeAgo from a date stored when pause/resume was clicked? This will always just display "Just Now" due to it being given a new date on every render.